src/tutorials/0008-merkle-dags/05.md

```diff
@@ -0,0 +1,82 @@
+---
+    title: "Merkle DAGs: Verifiability"
+    type: "multiple-choice" # can be "text", "code", "file-upload" or "multiple-choice"
+---
+
+Because we use cryptographic-strength hash algorithms to create
+CIDs, they offer a high degree of *verifiability*: an individual
+who retrieves data using a content address can always compute the
+CID for themselves to ensure that they got what they were after.
+This offers both permanence (the data behind a content address
+will never change) and protection against malicious manipulation
+(bad actors can’t trick you into downloading the wrong file
+without you recognizing that it’s not the file you requested).
+
+In a Merkle DAG, the CID of each node depends, in turn, on the
+CIDs of each of its children. As such, the CID of a root node
+uniquely identifies not just that node, but the *entire* DAG of
+which it’s the root! As a result, we can extend all of the
+awesome security, integrity, and permanence guarantees of CIDs to
+our data structure itself, and not just the data that it
+contains!
+
+Have you ever made a temporary backup copy of a file directory
+during some editing process, then found the two directories
+months later and wondered whether their contents were still
+identical? Rather than laboriously comparing files, you could
+compute a Merkle DAG for each copy: if the CIDs of the root
+directory matched, you’d know you could safely delete one and
+free up some space on your hard drive!
+
+## Any Node Can Be a Root Node
+
+DAGs can be seen as recursive data structures, each DAG
+consisting of *smaller* DAGs. In our example, the CID “baf...8”
+identifies a DAG, but so does the CID “baf...6”- it just
+identifies a smaller DAG, a subgraph of the one identified by
+“baf...8”. The corresponding nodes are *both* root nodes, given
+the right context.
+
+This is an extremely powerful and useful property. If we’re
+retrieving content that’s structured as a DAG, we don’t have to
+retrieve the entire DAG: we have the option of retrieving a
+subgraph instead, identified by its own top node’s CID (the top
+node of this subgraph would become its root node). If we want to
+share that subgraph with somebody else, we don’t have to include
+the context of the larger graph we originally retrieved the data
+from- we just have to send the CID of the subgraph. And, if we
+want to embed that subgraph as a part of a larger DAG that’s
+different from the one we originally found it in, we’re free to,
+because the DAG’s CID, which is the CID of its root node, depends
+the root node’s descendents and not its ancestors.
+
+That last point deserves greater emphasis: with Merkle DAGs, we
```

Comment:
Given that "forest" is usually thought of as consisting of discrete trees, and that we've avoided saying "tree" at all in this tutorial, so the metaphor/pun is a bit lost, we might reword the last sentence here.

Reply:
Elected for deletion.